cfpb/capital-framework
# Add brand-palette to cf-core and update default colors to use brand vars

Adds brand colors to `cf-core` and updates `cf-core` components to use the brand color variables.

See https://GHE/CFGOV/platform/issues/1064#issuecomment-124805 for details on why and how these changes are being made.

## Additions

- `brand-palette.less` is now part of `cf-core` component, so that we can use the color variables in this component and in others that depend on `cf-core`.

## Changes

- `cf-core` component uses these brand colors in `cf-vars.less` instead of the U.S. Web Design Standards hex values
- updates to `usage.md` documentation to reflect the brand color variables
- note that the link in `usage.md` for brand-palette.less doesn't work yet. it won't until this is merged to canary, published to npm, then merged to master.

## Testing

In this repo:
- check out this branch
- run `npm run cf-link`

In a second local clone of this repo in a different folder:

- pull down and check out the `gh-pages-rm-overrides` branch: `git fetch && git checkout gh-pages-rm-overrides`
- if you haven't already in this second clone, [run all the install steps for the project](https://github.com/cfpb/capital-framework/tree/gh-pages#installation)
- run `npm run cf-link`
- run `npm run build`
- run `npm start`
- Compare colors on local site: 
   - http://localhost:3000/components/cf-core/ 
   - to the live cf documentation to make sure they match: https://cfpb.github.io/capital-framework/components/cf-core/


## Review

- @user

## Screenshots


## Notes

-

## Todos

-

## Checklist
